DfE.Hexagonal.Common/Validation/ValidationResult.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DfE.Hexagonal.Common.Validation
+{
+    /// <summary>
+    /// contains a validation result and a dictionary of errors, if applicable
+    /// </summary>
+    public class ValidationResult
+    {
+        /// <summary>
+        /// is the thing valid?
+        /// </summary>
+        public bool IsValid { get; }
```

Comment:
Do short summary comments which simply restate the method name, add value? Is this something we wish to encourage?

Reply:
Just trying to think of a constructive suggestion.

What should this class "know" about? Thus, what is reasonable to reference within the comments?

For example, is `ValidationResult` intended to be independent of `IValidatable`? (i.e., loose coupling) 
If so, I hesitate to suggest tweaks such as `Is validation successful, per <c>IValidatable.Validate()</c>?` 

Alternatively, we could go the other way and make `ValidationResult` a nested class within `IValidatable`.

Reply:
To the short summary comment, I'd personally prefer consistency, i.e. everything has a summary. It might be short and repeating the obvious but the fact that it is there repeating the obvious means that you were right, it is obvious :) If someone forgets to put a summary or assumes that it is obvious when it isn't, that creates a problem.